style.ts

```diff
@@ -614,6 +618,10 @@ export interface LineSymbolizer extends BaseSymbolizer {
    * Renders the pixels of the line with a repeated pattern.
    */
   graphicFill?: PointSymbolizer;
+  /**
+   * Size of the additional area around the repeated graphic fill symbolizer.
+   */
+  graphicFillPadding?: [Expression<number>, Expression<number>, Expression<number>, Expression<number>];
```

Comment:
I don't know if this property (in `LineSymbolizer` make sens). I added it because we have also have a `graphicFill` property, but gapWidth and pattern, I think we can already do what we want here.

Reply:
I think it makes sense, but maybe only in two dimensions?

Reply:
Do you mean, here, in the `LineSymbolizer` That could make sens yes (`top/bottom`)

For the `FillSymbolizer` it's maybe possible but in my use case, but we currently have 4 differents value (top, right, bottom, left). and I would like to keep all 4.

Reply:
Yes the comment was only regarding the `LineSymbolizer`. I think the semantic of top/bottom or left/right depends on the direction of the line. I'd use something more general like "padding".